tools/webcam/camerad.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+import threading
+import os
+
+from cereal.visionipc import VisionIpcServer, VisionStreamType
+from cereal import messaging
+
+from openpilot.tools.webcam.camera import Camera
+from openpilot.common.realtime import Ratekeeper
+
+
+YUV_BUFFER_COUNT = int(os.getenv("YUV_BUFFER_COUNT", "20"))
+CAMERA_TYPE_STATE = {"roadCameraState":VisionStreamType.VISION_STREAM_ROAD,
+                    "driverCameraState":VisionStreamType.VISION_STREAM_DRIVER,
+                    "wideRoadCameraState":VisionStreamType.VISION_STREAM_WIDE_ROAD}
+
+def rk_loop(function, hz, exit_event: threading.Event):
+  rk = Ratekeeper(hz, None)
+  while not exit_event.is_set():
+    function()
+    rk.keep_time()
+
+class Camerad:
+  def __init__(self):
+    self.pm = messaging.PubMaster(['roadCameraState', 'driverCameraState', 'wideRoadCameraState'])
+    self.vipc_server = VisionIpcServer("camerad")
+
+    self.dual_camera = bool(int(os.getenv("DUAL","0")))
+    self.cameras, self.camera_threads = [], [] # ORDER: road_cam, driver_cam, wide_cam
```

Comment:
why does the order matter? it shouldn't

Reply:
It doesn't it was for me in the beginning. I will delete it